docs/markdown/snippets/gnome.md

```diff
@@ -0,0 +1,13 @@
+## `gnome.compile_schemas()` sets `GSETTINGS_SCHEMA_DIR` into devenv
+
+When using `gnome.compile_schemas()` the location of the compiled schema is
+added to `GSETTINGS_SCHEMA_DIR` environment variable when using
```

Comment:
It should be documented this can only be done once for a project.

If you have two different directories containing schema only one will work.

Reply:
Sorry, I was wrong, they can be multiple paths separated by `:`. TIL.

Reply:
> If you have two different directories containing schema only one will work.

Not anymore: https://gitlab.gnome.org/GNOME/glib/-/merge_requests/1315

Reply:
But should document that it requires GLib >= 2.64, if I read git log correctly.

Reply:
done.